decidim-module-census_connector/app/services/decidim/census_connector/person_proxy.rb

```diff
@@ -49,7 +49,7 @@ def census_qualified_id
       end
 
       def local_qualified_id
-        "#{user.id}@#{Decidim::CensusConnector.system_identifier}"
+        "#{user.id}@decidim"
```

Comment:
I think that in the future we could have two instances of Decidim connecting to the same Census instance. Also, I would like to use `participa` instead of `decidim` for this specific app.

I can't think an scenario where this is needed but, I don't know why, I feel that it could happen. We can wait until that moment, but then we would probably need to modify all users qualified ids, since it could be confusing to have an app using `decidim` identifier and the other not using it, while both apps would be Decidim instances.

Finally, another possible solution is using `Rails.application.class.parent.to_s.underscore` instead of a constant. This is a little bit simpler, but changing the application name could lead to unexpected behaviors.

Thougths?

Reply:
I don't really understand this configurability specially with census being multitenant. But if you feel it's necessary I can bring the settting back.

Reply:
Yes, you got a point there. Ok, let's leave it with your changes :+1: